entrypoint.sh

```diff
@@ -43,6 +43,9 @@ if [[ 1 -gt "$(cat $GITHUB_ENV | grep -c AWS_DEFAULT_REGION)" ]]; then
     echo "AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION" >> "$GITHUB_ENV"
 fi
 if [[ 1 -gt "$(cat $GITHUB_ENV | grep -c AWS_REGION)" ]]; then
-    AWS_REGION="${AWS_REGION:=$AWS_DEFAULT_REGION}"
-    echo "AWS_REGION=$AWS_REGION" >> "$GITHUB_ENV"
+    if [[ "$AWS_DEFAULT_REGION" != "$AWS_REGION" ]]; then
```

Comment:
shouldn't this already be true if we checked that AWS_REGION doesn't exist in GITHUB_ENV?